File: etl/models/proposta_governo.py
from sqlalchemy import Column, BigInteger, String, Boolean, Text, DateTime, ForeignKey
from .base import Base

class PropostaGoverno(Base):
    __tablename__ = "proposta_governo"

    sq_candidato = Column(BigInteger, ForeignKey("candidatura.sq_candidato"), primary_key=True)
    nm_arquivo = Column(String)
    ds_caminho_arquivo = Column(String)
    tx_conteudo_extraido = Column(Text)
    st_processado = Column(Boolean, default=False)
    dt_processamento = Column(DateTime)

File: src/backend/VoteBem/Entities/PropostaGoverno.cs
namespace VoteBem.Entities
{
    public class PropostaGoverno
    {
        public long SqCandidato { get; private set; }
        public string? NmArquivo { get; private set; }
        public string? DsCaminhoArquivo { get; private set; }
        public string? TxConteudoExtraido { get; private set; }
        public bool StProcessado { get; private set; }
        public DateTime? DtProcessamento { get; private set; }

        public Candidatura Candidatura { get; private set; } = null!;
        public ICollection<ResumoProposta> ResumosProposta { get; private set; } = [];

        protected PropostaGoverno() { }
    }
}
